native/app/types/response/book.ts

```diff
@@ -52,3 +52,22 @@ interface IUser {
   thumbnailUrl: string;
   username: string;
 }
+
+export interface IReviewResponse {
+    total: number;
+    offset: number;
+    limit: number;
+    reviews:IReview[];
+  };
+
+export interface IReview  {
+    createdAt: string;
+    book: {
+      id: number;
+      title: string;
+      thumbnailUrl: string;
+    };
+    impression: string;
+    id: number;
+    updatedAt: string;
+  }
```

Comment:
```suggestion
export interface IReview  {
  id: number;
  impression: string;
  book: {
    id: number;
    title: string;
    thumbnailUrl: string;
  };
  createdAt: string;
  updatedAt: string;
}
```

* インデントは揃える。
* (好みの問題だけど) フィールドの並びは、 createdAt, updatedAt は一番下に持ってきたい